scopes/react/react/templates/react-workspace-app/files/git-ignore.ts

```diff
@@ -0,0 +1,19 @@
+export function gitIgnore() {
```

Comment:
I think Gilad prefers the .gitignore that Github uses by default. You can see an example of this here: scopes/generator/generator/templates/workspace-generator/files/git-ignore-tpl.ts

Reply:
ok added common files folder so all reading the same git ignore and types

Reply:
great!